manifester/manifester.py

```diff
@@ -48,6 +52,24 @@ def access_token(self):
             ).json()
             self._access_token = token_data["access_token"]
         return self._access_token
+    
+    @property
+    def valid_sat_versions(self):
+        headers = {
+            "headers": {"Authorization": f"Bearer {self.access_token}"},
+            "proxies": self.manifest_data.get("proxies", settings.proxies),
+        }
+        valid_sat_versions = []
+        sat_versions_response = simple_retry(
+            requests.get,
+            cmd_args=[
+                    f"{self.allocations_url}/versions"
+                ],
+            cmd_kwargs=headers,
+            ).json()
+        for dict in sat_versions_response["body"]:
+            valid_sat_versions.append(dict["value"])
```

Comment:
best not to overwrite built-ins
```suggestion
        for ver_dict in sat_versions_response["body"]:
            valid_sat_versions.append(ver_dict["value"])
```

Reply:
Ack, thanks for pointing that out. Will make the change.